src/compute/src/render/mod.rs

```diff
@@ -1040,3 +1041,49 @@ where
         }
     })
 }
+
+thread_local! {
+    static SHARED_ROW:  std::cell::RefCell<Option<Row>> = std::cell::RefCell::new(None);
```

Comment:
Why use an `Option` here instead of starting directly from `Row::default()`?

Reply:
to enforce that `get` can only be called once

Reply:
I switched the logic to `Rc<RefCell<_>>` which seems to be more idiomatic.